docs/import/add_disaster_startup.js

```diff
@@ -26,4 +25,19 @@ authenticator.start();
 $('#create-new-disaster').on('click', () => toggleState(false));
 $('#cancel-new-disaster').on('click', () => toggleState(true));
 
+$('#tbody').sortable({
+  revert: true,
+  update: (event, ui) => updateAfterSort(ui),
+  // TODO: Table is still doing some jumping around on resort.
+  helper: function(e, tr) {
+    const originals = tr.children();
+    const helper = tr.clone();
+    helper.children().each(/* @this HTMLElement */ function(index) {
+      // Set helper cell sizes to match the original sizes
```

Comment:
nit: might be more helpful for this comment to be at the helper: ... line, and to explain what it's for:
"Keep each column in the dragged row the same width, even while it's being dragged" or similar?

Reply:
Done.